src/item.ts

```diff
@@ -0,0 +1,17 @@
+class Item {
+    public id: number;
+    public description: string;
+    public dueAt: Date;
+    public createdAt: Date;
+    public done: boolean;
```

Comment:
This is good but often I find booleans to not be super useful.  Often I replace them with Dates since not only do Dates act like booleans (set or not set) but they also tell you _when_ it happened.  This is more a personal preference but when it comes to DBs you may find these types of values to be much more userful than booleans for roughly the same space usage cost.